operations/client.go

```diff
@@ -20,6 +25,52 @@ func Client() cli.Command {
 	}
 }
 
+// RunKanopyOIDCLogin executes the kanopy-oidc login command and captures the JWT token from the output.
+// It also displays output to the user in real-time while ensuring that the JWT token is not printed.
+func RunKanopyOIDCLogin() (string, error) {
+	cmd := exec.Command("kanopy-oidc", "login", "-n", "-f", "device")
+
+	stdout, err := cmd.StdoutPipe()
+	if err != nil {
+		return "", errors.Wrap(err, "creating stdout pipe")
+	}
+
+	cmd.Stderr = os.Stderr // Show stderr output directly to the user
+
+	// Start the command
+	if err := cmd.Start(); err != nil {
+		return "", errors.Wrap(err, "starting kanopy-oidc login")
+	}
+
+	// Process the output - display non-token outputs and capture the token
+	var tokenBuf bytes.Buffer
+	scanner := bufio.NewScanner(stdout)
+
+	for scanner.Scan() {
+		line := scanner.Text()
+
+		// Check if the line contains a JWT (usually starts with "ey")
+		if len(line) > 30 && strings.HasPrefix(line, "ey") {
+			// This is likely the JWT token - capture but don't display
+			tokenBuf.WriteString(line)
+		} else {
+			// This is not a JWT token - display to the user
```

Comment:
isn't there a chance this could be the token still?

Reply:
I tested it locally and it was not